styles/components/section.css

```diff
@@ -0,0 +1,24 @@
+.section--styling {
```

Comment:
I think a better name would be just `section` as this is the block and not a modifier.